README.md

```diff
@@ -0,0 +1,190 @@
+# Charmed Data Integrator Operator
+
+## Overview
+
+This charm allows to retrieve product credentials needed to authenticate (SCRAM) with different kinds of data platform products([MongoDB](https://github.com/canonical/mongodb-operator), [MySQL](https://github.com/canonical/mysql-operator), [PostgreSQL](https://github.com/canonical/postgresql-operator), and [Kafka](https://github.com/canonical/kafka-operator)). 
+
```

Comment:
In the overview, we should try to convey the information as described in this document:

https://docs.google.com/document/d/1Zw-UoQzHMSQjATLXjjvd9GKu5qU2mHmqxtKck8nGkbU/edit